modules/microshift-cni-multus-bridge-nw-example-config.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-cni-multus-bridge-nw-example-config_{context}"]
+= Configuring a bridge-type additional network
+
+After you have created the NetworkAttachmentDefinition object and applied it, use the following example procedure to configure a bridge-type additional network. You can also use this workflow for other additional network types.
+
+.Prerequisite
+. You created and applied the NetworkAttachmentDefinition configuration.
+
+.Procedure
+. If it was not present before, verify that the bridge was created on the host by running the following command:
++
+[source,terminal]
+----
+$ ip a show br-test
+----
++
+.Example output
+[source,terminal]
+----
+22: br-test: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 96:bf:ca:be:1d:15 brd ff:ff:ff:ff:ff:ff
+    inet6 fe80::34e2:bbff:fed2:31f2/64 scope link
+       valid_lft forever preferred_lft forever
+----
+
+. Configure an IP address for the bridge by running the following command:
++
+[source,terminal]
+----
+$ sudo ip addr add 10.10.0.10/24 dev br-test
+----
+
+. Verify that the IP address configuration is added to the bridge by running the following command:
++
+[source,terminal]
+----
+$ ip a show br-test
+----
++
+.Example output
+[source,terminal]
+----
+22: br-test: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether 96:bf:ca:be:1d:15 brd ff:ff:ff:ff:ff:ff
+    inet 10.10.0.10/24 scope global br-test
+       valid_lft forever preferred_lft forever
+    inet6 fe80::34e2:bbff:fed2:31f2/64 scope link
+       valid_lft forever preferred_lft forever
+----
+
+. Verify the IP address of the pod by running the following command:
++
+[source,terminal]
+----
+$ oc get pod test-bridge --output=jsonpath='{.metadata.annotations.k8s\.v1\.cni\.cncf\.io/network-status}'
+----
++
+.Example output
+[source,terminal]
+----
+[{
+    "name": "ovn-kubernetes",
+    "interface": "eth0",
+    "ips": [
+        "10.42.0.17"
+    ],
+    "mac": "0a:58:0a:2a:00:11",
+    "default": true,
+    "dns": {}
+},{
+    "name": "default/bridge-conf",
+    "interface": "net1",
+    "ips": [
+        "10.10.0.20"
+    ],
+    "mac": "82:01:98:e5:0c:b7",
+    "dns": {}
+----
+
+. Optional. You can also `oc exec` into the pod and confirm its interfaces by using the `ip` command:
```

Comment:
Idk if this suggestion captures the right meaning, but want to avoid using a CLI phrase as a verb if possible
```suggestion
. Optional: You can use `oc exec` to access the pod and confirm its interfaces by using the `ip` command:
```